routers/authRouter.js

```diff
@@ -0,0 +1,56 @@
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const db = require("../data/helpers/user-model.js");
+const secret = process.env.SECRET;
```

Comment:
good that this can be declared at the top like an import, you can also give it a secondary value in case the .env file isn't found (which would be the case for production or for other devs to use your secret) like `const secret = process.env.SECRET || "Bruce Willis was a ghost the whole time"`